coordinates/sites.json

```
@@ -793,5 +793,91 @@
 	"longitude_unit": "degree",
 	"timezone": "US/Mountain",
 	"source": "NSO website: https://nsosp-dev.nso.edu/node/18"
+    },
+    "teide": {
+        "name": "Observatorio del Teide, Tenerife",
+        "elevation": 2390,
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 28.3,
+        "elevation_unit": "meter",
+        "longitude": 343.490278,
+        "timezone": "Atlantic/Canary",
+        "source": "Instituto de Astrofisica de Canarias website (http://research.iac.es/OOCC/observatorio-del-teide/ot/)",
+        "aliases": [
+            "Observatorio del Teide",
+            "OT"
+        ]
+    },
+    "CAHA": {
+        "name": "Centro Astronomico Hispano-Aleman, Almeria",
+        "elevation": 2168,
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 37.22361111,
+        "elevation_unit": "meter",
+        "longitude": 357.453889,
+        "timezone": "CET",
+        "source": "CAHA Wikipedia page (https://en.wikipedia.org/wiki/Calar_Alto_Observatory)",
+        "aliases": [
+            "Observatorio de Calar Alto",
+            "CAHA"
+        ]
+    },
+    "OARMA": {
+        "name": "Observatorio Astronomico Ramon Maria Aller, Santiago de Compostela",
+        "elevation": 240,
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 42.87594,
+        "elevation_unit": "meter",
+        "longitude": 351.44455,
+        "timezone": "CET",
+        "source": "OpenStreetView",
+        "aliases": [
+            "Observatorio Ramon Maria Aller",
+            "OARMA"
+        ]
+    },
+    "OAJ": {
+        "elevation": 1956,
+        "name": "Observatorio Astrofisico de Javalambre",
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 40.04201944,
+        "elevation_unit": "meter",
+        "longitude": 358.98391667,
+        "timezone": "CET",
+        "source": "Google Earth",
+        "aliases": [
+            "OAJ"
+        ]
+    },
+    "OAO": {
+        "name": "Okayama Astrophysical Observatory",
+        "elevation": 370,
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 34.57444444 ,
+        "elevation_unit": "meter",
+        "longitude": 133.59388889,
+        "timezone": "JST",
+        "aliases": [
+            "OAO"
+        ]
+    },
+    "TNO": {
+        "name": "Thai National Observatory",
+        "elevation": 2457,
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 18.59055556,
+        "elevation_unit": "meter",
+        "longitude": 98.48666667,
+        "timezone": "ICT",
```

```suggestion
        "timezone": "Asia/Bangkok",
        "source": "TNO Wikipedia page (https://en.wikipedia.org/wiki/Thai_National_Observatory)",
```
Coordinates confirmed there, although https://www.narit.or.th/index.php/thai-national-observatory-tno-location-menu has a location ~1' to the SW...